### ISSUE TITLE
Is there any theoretical reason why this could work?

### ISSUE THREAD

**Issue by laygr** [ZhengyaoJiang/PGPortfolio#71]
What anomaly could the agent be exploiting? Mean reversion, perhaps?

**Comment by DexHunter** [comment]
The agent is financial-model-free, you can read our paper on [arxiv](https://arxiv.org/pdf/1706.10059.pdf)

**Comment by laygr** [comment]
I’ve read your paper and I do recall saying it is model model-free, but still, the agent must be learning to identify momentum or mean reversion or maybe both, otherwise, it wouldn’t produce consistent returns, which, at least for the periods of training and testing used in your paper, it is being very consistent.

**Comment by laygr** [comment]
My guess is that the agent learnt a ratio between the cryptocurriencies which is mean reverting. I could do some tests to verify this theory. Please let me no if you are not exploring this idea, so that our work doesn’t overlap.

**Comment by DexHunter** [comment]
From my understanding, there are multiple patterns, but you can definitely draw some pictures (like the buying/selling signals) to see. Looking forward to your work.

**Comment by akaniklaus** [comment]
@laygr Yes although the agent is free to use any type of patterns, in general it acts very similar to mean-reversion based online portfolio selection methods (so it is more similar to follow-the-looser)

**Comment by HanLi123** [comment]
“Theory is when you know everything but nothing works. Practice is when everything works but no one knows why. In our lab, theory and practice are combined: nothing works and no one knows why.”
